iothub/service/src/DeviceGroupProperties.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Microsoft.Azure.Devices
+{
+    using System;
+    using Newtonsoft.Json;
+    #pragma warning disable 1591
+    public class DeviceGroupProperties
+    {
+
+        //
+        // Summary:
+        //     The flag to mark deleted records.
+        [JsonProperty(PropertyName = "deleted", NullValueHandling = NullValueHandling.Ignore)]
+        public bool Deleted { get; set; }
+        //
+        // Summary:
+        //     The version of the query language used in the query expression. TODO: links to
+        //     queryVersion documentation.
+        [JsonProperty(PropertyName = "queryVersion", NullValueHandling = NullValueHandling.Ignore)]
+        public string QueryVersion { get; set; }
```

Comment:
Should there be a default version assigned? Perhaps not but the doc comments remarks could imply a good default.